src/ZODB/tests/testConnection.py

```diff
@@ -521,29 +529,18 @@ def doctest_invalidate(self):
         Transaction ids are 8-byte strings, just like oids; p64() will
         create one from an int.
 
-        >>> cn.invalidate(p64(1), {p1._p_oid: 1})
+        >>> mvcc_storage.invalidate(p64(1), {p1._p_oid: 1})
```

Comment:
These were used internally by ZODB. Nothing should depend on them externally. They never should have been in IConnection.  I'd rather just consider their inclusion in IConnection a bug that this PR fixes.